libxrandr will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxrandr it will fail in the following way:

```
2024/11/12 23:12:54 INFO running step "test/pkgconf"
2024/11/12 23:12:54 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:54 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:54 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:54 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:54 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + basename /home/build/melange-out/libxrandr-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + dev_pkg=libxrandr-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + apk info -L libxrandr-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + grep -q ^Name: usr/lib/pkgconfig/xrandr.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + basename usr/lib/pkgconfig/xrandr.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + lib_name=xrandr uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + echo usr/lib/pkgconfig/xrandr.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + grep -q '^usr/lib/pkgconfig/xrandr.pc$\|^usr/share/pkgconfig/xrandr.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + pkgconf --exists xrandr uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + grep -q ^Version: usr/lib/pkgconfig/xrandr.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + pkgconf --modversion xrandr uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 INFO 1.5.4 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + grep -q ^Libs: usr/lib/pkgconfig/xrandr.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + pkgconf --libs xrandr uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 INFO -lXrandr uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + grep -q ^Cflags: usr/lib/pkgconfig/xrandr.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN + pkgconf --cflags xrandr uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN Package x11 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN Perhaps you should add the directory containing `x11.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN Package 'x11', required by 'xrandr', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN Package 'x11', required by 'xext', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 WARN Package 'xrender', required by 'xrandr', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:54 INFO pod 36d0562e9b382cff16ab88dd3702454f67718c3db7424b32a6af409e763c1139 terminated
2024/11/12 23:12:54 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:54 INFO   workspace dir: /home/user/tmp/melange-workspace-3377598589
2024/11/12 23:12:54 INFO   guest dir: /home/user/tmp/melange-guest-3589502094
2024/11/12 23:12:54 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxrandr] Error 1
```

--- a/libxrandr.yaml
+++ b/libxrandr.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxrandr
   version: 1.5.4
-  epoch: 1
+  epoch: 2
   description: X11 RandR extension library
   copyright:
     - license: MIT
@@ -40,9 +40,14 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - libx11-dev
         - libxrandr
         - libxext-dev
+        - xorgproto
     description: libxrandr dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxrandr-doc
     pipeline:

